build_scripts/build_dependencies_windows_no_cuda.ps1

```diff
@@ -14,4 +36,9 @@ pip3 install meson
 # Install cmake
 pip3 install cmake --upgrade
 
-Write-Host "Dependencies verified and installed successfully."
\ No newline at end of file
+Write-Host "Dependencies verified and installed successfully."
+
+# Build Documentation
```

Comment:
We Should not build documentation while building dependencies

Reply:
Removed documentation build from scripts.